### PR TITLE
#165362028 implement feedback changes

### PR DESCRIPTION
**Pull request by Tunji17**
### What does this PR do?
IT restructures my server folder to include Controllers, Routes, Middleware, Models, helpers, and my app.js, converted my controllers to classes and used import/export 
#### Description of Task to be completed?
Restructure server, convert functions to es6 classes, make use of import/export instead of requiring
#### How should this be manually tested?
When you clone the repo, set the NODE_ENV=development and run npm run build to build into the dist folder, then run npm run devstart to start the server with nodemon
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165362028](https://www.pivotaltracker.com/story/show/165362028)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Comment by kampkelly** [comment]
Your "how should this be manually tested" should include details on how I can run the project if I were to clone it. Please update it.

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
-tests/
```
Comment (kampkelly) [inline .eslintignore]: add a new line

**Comment by kampkelly** [comment]
Your commit messages don't appear to have a body. Can you amend them.

File: server/app.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-const helmet = require('helmet');
-const path = require('path');
-const cookieParser = require('cookie-parser');
-const swaggerUi = require('swagger-ui-express');
-const logger = require('./logger');
-const { sendJSONResponse } = require('./server/helpers');
-const docs = require('./docs');
+// require('dotenv').config({ path: '../.env' });
+// import _ from './env';
+// import dotenv from 'dotenv';
```
Comment (kampkelly) [inline server/app.js]: Remove these comments.

File: server/app.js [hunk 3]
```diff
@@ -21,8 +25,8 @@ global.db = {
   },
 };
 
-// const appRoutes = require('./server/routes/appRoutes');
-const apiRoutes = require('./server/routes/apiRoutes');
+// const appRoutes = require('./routes/appRoutes');
+const apiRoutes = require('./routes/apiRoutes');
```
Comment (kampkelly) [inline server/app.js]: remove comments here also

Comment (Tunji17) [reply server/app.js]: okay

**Pull request merged**